sample/src/main/java/com/schibsted/spain/barista/sample/FlakyActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.schibsted.spain.barista.sample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+public class FlakyActivity extends AppCompatActivity {
```

Comment:
@Sloy what do you think about calling this Activity HelloWorldActivity? Actually, this Activity is not flaky at all, so we could use it for other tests, to avoid creating Activities everytime we need to test another feature in Barista. In addition, we could use this HelloWorldActivity to test that the text appears (but that's for other PR)